docs/03-github/04-builder.mdx

```diff
@@ -454,6 +454,30 @@ to avoid any conflicts.
 
 _**required:** `false`_ _**default:** `"/github/workspace"`_
 
+#### dockerCpuLimit
+
+Number of CPU cores to assign the Windows docker container. Defaults to all available cores when no
+value is specified. Can accept fractional values, ie 0.5 for half of a cpu core's execution time.
+
+_**required:** `false`_ _**default:** `""`_
+
+#### dockerMemoryLimit
+
+Amount of memory to assign the docker container. Defaults to 95% of total system memory rounded down
+to the nearest megabyte on Linux and 80% on Windows when no value is specified. On unrecognized
+platforms, defaults to 75% of total system memory. To manually specify a value, use the format
+\<number\>\<unit\>, where unit is either m or g. ie: 512m = 512 megabytes and 4g = 4 gigabytes.
+
+_**required:** `false`_ _**default:** `""`_
+
+#### dockerIsolationMode
+
+Isolation mode to use for the docker container. Can be one of process, hyperv, or default. Default
```

Comment:
Same nitpick about consistency for nitpick: "Windows docker container" vs "docker container"